### PR TITLE
Jmac/pylint fixes 2

### PR DESCRIPTION
**Pull request by jmacarthur**
Another batch of fixes suggested by PyLint warnings.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -126,13 +126,14 @@ $(PIP):
 PYLINT := pipenv run pylint
 PYCODESTYLE := pipenv run pycodestyle
 PYDOCSTYLE := pipenv run pydocstyle
+PYLINTOPTIONS := --load-plugins=pylint.extensions.bad_builtin
```
Comment (jacebrowning) [inline Makefile]: Can this be specified in `.pylint.ini`?

File: doorstop/core/tests/test_all.py [hunk 2]
```diff
@@ -141,6 +142,7 @@ def test_issues_skipped_level(self):
         expect = DoorstopInfo("skipped level: 1.4 (REQ003), 1.6 (REQ004)")
         for issue in self.document.issues:
             logging.info(repr(issue))
+            # pylint: disable=C0123
```
Comment (jacebrowning) [inline doorstop/core/tests/test_all.py]: I prefer to see these warnings disabled by name rather than ID.

**Comment by jacebrowning** [comment]
Sorry to see that we're stepping on each other's toes, but I was able to get rid of the rest of the warnings with my PR from yesterday: https://github.com/jacebrowning/doorstop/pull/258

Would you mind taking a look at that one?

**Comment by jmacarthur** [comment]
No problem, I'll review that one, just been a bit busy today.

**Comment by jacebrowning** [comment]
Thanks! I also added you as a collaborator so you can create branches directly on this repository.

**Comment by jmacarthur** [comment]
Superceded by #258.

**Pull request closed without merging**